Index: .gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>/.env\n/.venv/\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.gitignore b/.gitignore
--- a/.gitignore	(revision 5b46996636f81e9e35edf937e150dbfbb2600e6b)
+++ b/.gitignore	(date 1732812915565)
@@ -1,2 +1,3 @@
 /.env
 /.venv/
+/.idea/
